Scripts/BuildDatabank/AddData.py

```diff
@@ -23,20 +23,19 @@
 pd.set_option("display.max_colwidth", 1000)
 
 from MDAnalysis import Universe
-from OrderParameter import *
 
+sys.path.append('..')
+from DatabankLib.OrderParameter import *
```

Comment:
to have an internal consistency in importing databank related modules and methods, could you please explicitly define the imported methods here, instead of using *?

Reply:
done during style alignment.